cpp/ql/src/semmlecode.cpp.dbscheme

```diff
@@ -408,7 +408,22 @@ function_deleted(unique int id: @function ref);
 
 function_defaulted(unique int id: @function ref);
 
+/**
+ * Holds if the function is declared to be `constexpr`.
+ */
+function_declared_constexpr(unique int id: @function ref);
```

Comment:
Can't we use the `specifiers` table for this? That's where we're currently storing the related specifiers like `const`, `volatile`, `inline`, etc.